dts/bindings/serial/zephyr,uart-bridge.yaml

```
@@ -0,0 +1,23 @@
+# Copyright 2025 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  UART bridge
+
+  Bridges data from two serial devices, for example a USB CDC-ACM serial port
+  and an hardware UART.
```

```suggestion
title: UART bridge

description: |
  Bridges data from two serial devices, for example a USB CDC-ACM serial port
  and an hardware UART.
```
New keyword has merged ;)

---

fancy, done